src/components/20-molecules/dropdown/index.js

```diff
@@ -103,15 +103,16 @@ class AXADropdown extends NoShadowDOM {
 
   set value(newValue) {
     const {
-      state: { isControlled, value },
+      state: { isControlled, value, firstTime },
     } = this;
     // first value coming in indicates controlledness?
-    if (!isControlled && newValue !== undefined) {
+    if (!isControlled && newValue !== undefined && firstTime) {
       // yes, remember in model state
       this.state.isControlled = true;
     }
     // update state
     this.state.value = newValue;
+    this.state.firstTime = false;
```

Comment:
this get state set firstTime -> `this get state` is duplicated more then once. Please do something like this:

`const { state } = this`
`state.isControlled `
`state.value`
`state.firstTime `

Reply:
I did something equivalent, but simpler...